utils/prometheus/Makefile

```diff
@@ -42,22 +42,42 @@ $(call Package/prometheus/Default)
   CATEGORY:=Utilities
 endef
 
+define Package/prometheus-web-console
+$(call Package/prometheus/Default)
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=prometheus
+endef
+
 define Package/prometheus/description
 Prometheus, a Cloud Native Computing Foundation project, is a systems and
 service monitoring system. It collects metrics from configured targets at given
 intervals, evaluates rule expressions, displays the results, and can trigger
 alerts if some condition is observed to be true.
 endef
 
+define Package/prometheus-web-console/description
+Web console for the Prometheus monitoring service.
+endef
+
 define Package/prometheus/install
 	$(call GoPackage/Package/Install/Bin,$(1))
 
 	$(CP) ./files/* $(1)/
 endef
 
+define Package/prometheus-web-console/install
+	$(INSTALL_DIR) $(1)/usr/share/prometheus/consoles
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/consoles/* $(1)/usr/share/prometheus/consoles/
```

Comment:
consoles/ contains .html files, they don't have to have 755 permissions

Reply:
fair, this could be INSTALL_DATA